cves/kernel/CVE-2020-9383.yml

```diff
@@ -75,7 +78,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: ["https://bugzilla.redhat.com/show_bug.cgi?id=1810685"]
```

Comment:
I think you should list the IDs of the bugs involved. In this case:
 [1810687, 1815403, 1815404, 1815405, 1888662]

Reply:
Thank you for catching these as well! Similar to the comment on the other file, I can only access/find the first that you've listed so I'll add that